test/test-cases/functional/saic/test_vm_to_vm_commn_acl_inbound.py

```diff
@@ -0,0 +1,179 @@
+import json
+from pathlib import Path
+from pprint import pprint
+import time
+import pytest
+import sys
+sys.path.append("../utils")
+import snappi_utils as su
+
+current_file_dir = Path(__file__).parent
+
+"""
+This covers following scenario :
+
+Test vnet to vnet communication with ACL on inbound direction:
+1. Configure DUT to deny and allow traffic
+2. Configure TGEN traffic flow as one vnet to another vnet of two OpenTrafficGenerator ports
+3. Verify Traffic denied through deny IPs
+
+
+Topology Used :
+
+       --------          -------          -------- 
+      |        |        |       |        |        |
+      |        |        |       |        |        |
+      |  TGEN  |--------|  DUT  |--------|  TGEN  |
+      |        |        |       |        |        |
+      |        |        |       |        |        |
+       --------          -------          -------- 
+       
+"""
+
+###############################################################
+#                  Declaring Global variables
+###############################################################
+
+TOTALPACKETS = 1000
+PPS = 100
+TRAFFIC_SLEEP_TIME = (TOTALPACKETS / PPS) + 2 
+PACKET_LENGTH = 128
+ENI_IP = "1.1.0.1"
+NETWORK_IP2 = "1.128.0.2"
+NETWORK_IP1 = "1.128.0.1"
+
+DPU_VTEP_IP = "221.0.0.2"
+ENI_VTEP_IP = "221.0.1.11"
+NETWORK_VTEP_IP = "221.0.2.101"
+
+###############################################################
+#                  Start of the testcase
+###############################################################
+
+
+@pytest.mark.xfail(reason="https://github.com/sonic-net/DASH/issues/345")
+class TestAclInbound:
+    @pytest.fixture(scope="class")
+    def setup_config(self):
+        """
+        Fixture returns the content of the file with SAI configuration commands.
+        scope=class - The file is loaded once for the whole test class
+        """
+        current_file_dir = Path(__file__).parent
+        with (current_file_dir / 'config_inbound_setup_commands.json').open(mode='r') as config_file:
```

Comment:
To improve readability you can do one of two things:

OPTION no.1:
Use variables in the JSON configuration like `{DPU_VTEP_IP}` and when load it do the following
```
config = json.loads(open(file_path).read().format(**globals()))
```

OPTION no.2:
Convert configs to *.py and move there all constants and import this config.py in the test case.

*How it can help?*
- You won't need to keep in track test's code and config files to have same IPs, etc.
- Both file will use the same variable names and it will improve readability.

Reply:
This is breaking the test.. Will take care of this in the next cases

Reply:
Why? You are not changing the content, just loading it in a different way.